css/style_index.css

```diff
@@ -12,6 +12,7 @@ a {
 
 button:hover {
 	cursor: pointer;
+	font-weight: bold;
```

Comment:
According to me this property 'button:hover' is unused, it must be my mistake. So adding font-weight would not affect anything, in fact it should be deleted completely because it's useless. So if you willing to contribute then you can remove this 'button:hover' totally or can find any other improvement which would be helpful.

Reply:
Meanwhile you can also contact me on linkedIn or by email for any other queries
- shivanisingh260500@gmail.com
- https://www.linkedin.com/in/shivani-singh-86b640192/